pkg/controller/statusmanager/pod_status.go

```diff
@@ -120,17 +128,46 @@ func (status *StatusManager) setNodeNetworkUnavailable(nodeName string, ready bo
 		Reason:             reason,
 		Message:            message,
 	}
-	SetNodeCondition(&node.Status.Conditions, condition)
-	err = status.client.Status().Update(context.TODO(), node)
+	changed := SetNodeCondition(&node.Status.Conditions, condition)
+	if !changed {
+		log.Info("Node condition is not changed")
+		return
+	}
+	if !ready {
+		status.updateNodeStatus(node)
+		return
+	}
+	last := time.Now()
+	_, present := sharedInfo.LastNetworkAvailable[nodeName]
+	if !present {
+		sharedInfo.LastNetworkAvailable[nodeName] = last
+		log.Info(fmt.Sprintf("Last network unavailable time of node %s is set to %s", nodeName, last))
+
+		go func() {
+			start := time.Now()
```

Comment:
`start` is only used for the logging below, is it the goroutine name? I mean can we remove `start`?

Reply:
absolutely yes, I just used it for marking the goroutine and debugging. done